### ISSUE TITLE
‘Aborted’ log encountered at the end of onnx2trt  conversion

### ISSUE THREAD

**Issue by syboomsy** [NVIDIA/TensorRT#2824]
## Description

Converted a mask-rcnn onnx model into TRT format,  though  serialized trt engine got successfully,
a line of abortion message is shown in the terminal as : 
![git_issue_0](https://user-images.githubusercontent.com/57123221/228193842-ca5dd855-3659-42bd-9034-06f498b7995f.png)

I am quite a novice of trt-based model deployment,  so could someone tell me the possible cause  the log, does it indeed indicate some internal errors during  the conversion?

The conversion command is as follows:
`trtexec 
    --onnx=/mapper_dir/user/TensorRT/samples/python/updated_model_bak.onnx 
    --saveEngine=/mapper_dir/user/TensorRT/samples/python/test2.trt 
    --minShapes=image:1x240x320x3 
    --optShapes=image:1x480x640x3 
    --maxShapes=image:1x480x640x3 
    --verbose`

## Environment

**TensorRT Version**:  8.6.0 EA
**NVIDIA GPU**:  RTX 3090 24G
**NVIDIA Driver Version**:  460.84
**CUDA Version**:  11.2
**CUDNN Version**:  8.8
**Operating System**:  ubuntu 20
**Python Version (if applicable)**:  3.8
**Tensorflow Version (if applicable)**:  
**PyTorch Version (if applicable)**:  1.9
**Baremetal or Container (if so, version)**:  container 20.10.16

**Comment by syboomsy** [comment]
Again when I modify the convert command option "--maxShapes=image:1x480x640x3" into "--maxShapes=image:10x480x640x3" , the last line log of conversion changes to "Segmentation fault (core dumped) ",

**Comment by zerollzeng** [comment]
Can you share the onnx here? Thanks!

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, pls reopen if you still have question. thanks!
